File: swagger-codegen/csharp-dotnet2/src/main/CsharpDotNet2/samsara/Model/VehicleMaintenancePassengerCheckEngineLight.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace samsara.Model {

  /// <summary>
  /// Passenger vehicle check engine light.
  /// </summary>
  [DataContract]
  public class VehicleMaintenancePassengerCheckEngineLight {
    /// <summary>
    /// Gets or Sets IsOn
    /// </summary>
    [DataMember(Name="isOn", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "isOn")]
    public bool? IsOn { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class VehicleMaintenancePassengerCheckEngineLight {\n");
      sb.Append("  IsOn: ").Append(IsOn).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: openapi-generator/clojure/src/samsara_api/specs/vehicle_maintenance_passenger_check_engine_light.clj
(ns samsara-api.specs.vehicle-maintenance-passenger-check-engine-light
  (:require [clojure.spec.alpha :as s]
            [spec-tools.data-spec :as ds]
            )
  (:import (java.io File)))


(def vehicle-maintenance-passenger-check-engine-light-data
  {
   (ds/opt :isOn) boolean?
   })

(def vehicle-maintenance-passenger-check-engine-light-spec
  (ds/spec
    {:name ::vehicle-maintenance-passenger-check-engine-light
     :spec vehicle-maintenance-passenger-check-engine-light-data}))
